packages/teleport-project-generator/src/index.ts

```diff
@@ -192,6 +195,30 @@ export class ProjectGenerator {
       injectFilesToPath(rootFolder, this.strategy.components.path, [componentsModuleFile])
     }
 
+    // Handling framework specific changes to the project
+    const { framework } = this.strategy
+
+    if (framework && framework.config) {
+      const { fileName, fileType, styleVariation } = framework.config
```

Comment:
just a thought when seeing this, can we maybe think of a way to create plugins for project generators? these plugins would do these kind of string changes, config updates, etc.

Reply:
Yeah, sounds interesting